app/src/main/java/ar/scacchipa/twittercloneapp/fragment/FragmentHappyPath.kt

```diff
@@ -0,0 +1,33 @@
+package ar.scacchipa.twittercloneapp.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.navArgs
+import ar.scacchipa.twittercloneapp.databinding.FragmentHappyPathBinding
+
+class FragmentHappyPath : Fragment() {
```

Comment:
maybe better naming would be nice.

Reply:
The identifier of the fragment was changed to FragmentLoginSuccess

Reply:
The identifier of the fragment was changed to FragmentHome